src/pages/security/static/_vulnerabilities-types-content.ts

```diff
@@ -0,0 +1,101 @@
+import LowIcon from 'images/svg/security/low.svg'
+import MediumIcon from 'images/svg/security/medium.svg'
+import HighIcon from 'images/svg/security/high.svg'
+import CriticalIcon from 'images/svg/security/critical.svg'
+
+export const vulnerabilitiesTypesContent = [
```

Comment:
```suggestion
export const vulnerability_types_content = [
```
should be in snake case.